### ISSUE TITLE
BEAT2 Pose parameters

### ISSUE THREAD

**Issue by fluide1022** [PantoMatrix/PantoMatrix#133]
Wonderful work!
I have noticed the "poses" parameter dimension is 165. So what does it contain?
"body_pose", "jaw_pose" or something else?

**Comment by H-Liu1997** [comment]
Thank you for the positive feedback!

Consider we have 55 joints,
1-22 body joints
23 jaw pose
24 and 25 left and right eye
26-55 left and right hands

For each joint we represent it via 3d axis angle, then finally 165 numbers

Feel free to let me know if you have any other questions

**Comment by fluide1022** [comment]
Thanks for your reply. How do you get these parameters? I tried the SMPLer-X method(SMPLer-X: Scaling Up Expressive Human Pose and Shape Estimation), but it doesn't work in the blender smplx addon.

**Comment by H-Liu1997** [comment]
We use mosh++ to get these parameters, if you save the SMPLer-X results in
the same npz format, will it work?

**Comment by fluide1022** [comment]
I think the SMPLer-X keypoint order is not same with the mosh++

**Comment by sunday9999** [comment]
@fluide1022 Have you solved the problem? I encountered the same problem
@H-Liu1997 
Is the[ blender smplx addon](https://drive.google.com/drive/folders/1ukbifhHc85qWTzspEgvAxCXwn9mK4ifr) you provided the official version? Or did you modify it yourself?

**Comment by H-Liu1997** [comment]
Hello, if your npz file works on previous addon it will work fine on addon
we provided.

Kindly remind please use blender 3.3, it will not work on blender 2.93.

May I ask will your npz file work on previous version addon you download
from MPI?

**Comment by sunday9999** [comment]
I suspect that this issue is related to the initial state of the hand poses.

From the [video](https://www.youtube.com/watch?v=T0OYPvViFGE) you shared, the hand poses appear in a specific way. When opening this action in Blender, the initial state of the hand poses should be flat. 
in your video 
![image](https://github.com/user-attachments/assets/f52cbe65-b924-4949-9ecd-b4faf6a3c9c3)

blender
![image](https://github.com/user-attachments/assets/7f04c18f-f91b-493b-bd5e-7323f4fd34da)

init hand post : flat
![image](https://github.com/user-attachments/assets/8c1c3c70-8f26-4d6b-abb1-6f9ce578056e)

init hand pose: relaxed
![image](https://github.com/user-attachments/assets/52ee8b5e-8929-41b7-b4a8-7997573e0bc8)



However, the smplx.create function has a parameter **flat_hand_mean**, which defaults to false. If this parameter is not specified, the default hand pose will not be flat, as referenced in the content of this link:https://github.com/vchoutas/smplx/issues/124.

`class SMPLH(SMPL):

    def __init__(
        self, 
        ...
        flat_hand_mean: bool = False,  #  flat_hand_mean: bool, optional  If False, then the pose of the hand is initialized to False.
       ....
    ) -> None:
  


I am confused because I see that flat_hand_mean is not set in the code. Does this mean that the hand poses during training were incorrect? The demo.py provided by the code also renders the hands in a relaxed state (you can see that the ground truth rendered by the old code is also incorrect).

**Comment by fluide1022** [comment]
> @fluide1022 Have you solved the problem? I encountered the same problem @H-Liu1997 Is the[ blender smplx addon](https://drive.google.com/drive/folders/1ukbifhHc85qWTzspEgvAxCXwn9mK4ifr) you provided the official version? Or did you modify it yourself?
Yes. I have solved this issue. After using the human estimation for the human image, you need to adjust the SMPLX trans parameters and other necessary parameters
